scheduler/test/cook/test/mesos/monitor.clj

```diff
@@ -196,7 +201,9 @@
     (is (= 0 (counter ["hungry" "users"])))
     (is (= 0 (counter ["satisfied" "users"])))
 
-    (api/create-jobs! conn {::api/jobs [job3]})
+    (with-redefs
+      [job-submission-rate-limiter ratelimit/NOOP-RateLimiter]
+      (api/create-jobs! conn {::api/jobs [job3]}))
```

Comment:
Why not use the wrapper helper function?

Reply:
See explanation above.